skimage/io/_plugins/matplotlib_plugin.py

```diff
@@ -49,7 +51,7 @@ def _get_image_properties(image):
     out_of_range_float = (np.issubdtype(image.dtype, np.float) and
                           (immin < lo or immax > hi))
     low_dynamic_range = (immin != immax and
-                         (float(immax - immin) / (hi - lo)) < (1. / 255))
+                         is_low_contrast(image))
```

Comment:
This reminds me: maybe `is_low_contrast` should also return `False` when `immin == immax`? The idea here is that a low-contrast warning will let people know that if they don't see any variation in the image, that's the probable reason. But if there *really is* no variation in the image, perhaps the warning is not warranted?